### PR TITLE
Centrys 51

### PR DESCRIPTION
**Pull request by centrys-george**

### CODE REVIEW

File: libs/transactionProcessor.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    this.sendTransactionCall = (transactionData, withholdPercent, trySend) => {
+        return function (callback) {
+            const transactionDetails = {};
+            transactionDetails.txHash = -1;
+            transactionDetails.to = transactionData.to;
+            transactionDetails.amount = transactionData.value;
+
+            unlockAccountIfNecessary(transactionData.from, poolOptions.addressPassword, function (isUnlocked) {
+                if (isUnlocked) {
+                    logger.debug(logSystem, logComponent, "Sending " + transactionData.value / magnitude + " AION to " + transactionData.to);
```
Comment (alinturbut) [inline libs/transactionProcessor.js]: Please use minerRewardsLogger

**Comment by alinturbut** [comment]
Remove tests/ and libs/logUtil.js

File: libs/paymentProcessor.js [hunk 2]
```diff
@@ -363,31 +377,37 @@ function SetupForPool(logger, minersRewardLogger, poolOptions, setupFinished) {
...
-                        sendTransactionCalls.push(sendTransactionCall(transactionData, withholdPercent, addressAmounts, totalSent, trySend));
+                        sendTransactionCalls.push(transactionProcessor.sendTransactionCall(transactionData, withholdPercent, addressAmounts, totalSent, trySend));
                     }
 
                     if (sendTransactionCalls.length > 0) {
-                       getTransactionDatasForPoolOps(totalRewardBeforeWitholds).forEach(function (element) {
-                           sendTransactionCalls.push(sendTransactionCall(element, withholdPercent, addressAmounts, totalSent, trySend));
-                       });
+                        getTransactionDatasForPoolOps(totalRewardBeforeWitholds).forEach(function (element) {
+                            sendTransactionCalls.push(sendTransactionCall(element, withholdPercent, trySend));
```
Comment (alinturbut) [inline libs/paymentProcessor.js]: transactionProcessor.sendTransactionCall

**Comment by centrys-george** [comment]
This will solve #23

**Review by alinturbut — APPROVED** [review]

**Pull request merged**